operators/multiclusterobservability/pkg/rendering/renderer_mcoa.go

```diff
@@ -164,6 +170,121 @@ func (r *MCORenderer) renderClusterManagementAddOn(
 	}
 	u.SetLabels(cLabels)
 
+	cma := &addonapiv1alpha1.ClusterManagementAddOn{}
+	if err := runtime.DefaultUnstructuredConverter.FromUnstructured(u.Object, cma); err != nil {
+		return nil, err
+	}
+
+	if r.cr.Spec.Capabilities != nil {
+		if len(cma.Spec.InstallStrategy.Placements) != 1 {
+			return nil, fmt.Errorf("expected exactly one placement, got %d", len(cma.Spec.InstallStrategy.Placements))
+		}
+		globalConfigs := []addonapiv1alpha1.AddOnConfig{}
+		if r.cr.Spec.Capabilities.Platform != nil && r.cr.Spec.Capabilities.Platform.Metrics.Collection.Enabled {
+			globalConfigs = append(globalConfigs, []addonapiv1alpha1.AddOnConfig{
+				{
+					ConfigGroupResource: addonapiv1alpha1.ConfigGroupResource{
+						Group:    prometheusalpha1.SchemeGroupVersion.Group,
+						Resource: prometheusalpha1.PrometheusAgentName,
+					},
+					ConfigReferent: addonapiv1alpha1.ConfigReferent{
+						Name:      "acm-platform-metrics-collector-default",
+						Namespace: mcoconfig.GetDefaultNamespace(),
+					},
+				},
+				{
+					ConfigGroupResource: addonapiv1alpha1.ConfigGroupResource{
+						Group:    prometheusalpha1.SchemeGroupVersion.Group,
+						Resource: prometheusalpha1.ScrapeConfigName,
+					},
+					ConfigReferent: addonapiv1alpha1.ConfigReferent{
+						Name:      "platform-metrics-default",
+						Namespace: mcoconfig.GetDefaultNamespace(),
+					},
+				},
+				{
+					ConfigGroupResource: addonapiv1alpha1.ConfigGroupResource{
+						Group:    prometheusv1.SchemeGroupVersion.Group,
+						Resource: prometheusv1.PrometheusRuleName,
+					},
+					ConfigReferent: addonapiv1alpha1.ConfigReferent{
+						Name:      "platform-rules-default",
+						Namespace: mcoconfig.GetDefaultNamespace(),
+					},
+				},
```

Comment:
My aim is to have it working out of the box for the base case. It lists here the metrics we need to collect for having the platform dashboards working. What are you thinking about?

Reply:
The issue is that we must let the user the ability to tune this list himself. As of now, he must be able to remove metrics from this list. However, the need for most or our users is to collect more metrics.
This is something I must discuss again with our PM. Maybe this will change later on.

Reply:
I could probably remove the rules more easily however. I'll think about it

Reply:
If there I a need I'm not opposed to having them here I was just double checking 👍

Reply:
I'm leaving it like this for now. I might remove some of them after reviewing the list of needed metrics, in another PR.